site/sfguides/src/call_centre_analytics_with_snowflake_cortex_and_spcs/call_centre_analytics_with_snowflake_cortex_and_spcs.md

```diff
@@ -0,0 +1,171 @@
+author: sfc-gh-praj
```

Comment:
it's best to add your name instead of GH username

Reply:
Added the author name